spaces/dego/skin.scss

```
@@ -0,0 +1,9 @@
+.name {
```

The css selector should similar to the space key.
```suggestion
.dego {
```